src/display.hpp

```diff
@@ -1026,6 +1026,7 @@ class display : public video2::draw_layering
 	reach_map reach_map_old_;
 	bool reach_map_changed_;
 	void process_reachmap_changes();
+	std::set<map_location> units_that_can_reach_goal_;
```

Comment:
Given that this variable is only used in `game_display`, it should be defined there.

Reply:
Changed.